frontend/src/components/camp/Picasso.vue

```diff
@@ -216,7 +205,7 @@ export default {
     },
     scheduleEntryRoute,
     showScheduleEntry (entry) {
-      this.$router.push(scheduleEntryRoute(this.camp, entry))
+      this.$router.push(scheduleEntryRoute(this.camp, entry)).catch(() => {})
```

Comment:
Prevents `NavigationDuplicated` error when clicking on the same activity in the SideBar calendar view, see https://github.com/vuejs/vue-router/issues/2872.